miasm2/jitter/emulatedsymbexec.py

```diff
@@ -92,9 +92,13 @@ def _simp_handle_segm(self, e_s, expr):
             return expr
         segm_nb = int(expr.args[0].arg)
         segmaddr = self.cpu.get_segm_base(segm_nb)
-        return e_s(m2_expr.ExprOp("+",
-                                  m2_expr.ExprInt(segmaddr, expr.size),
-                                  expr.args[1]))
+	
+	arch_size = self.ir_arch.attrib
+	expr.args[1]._size = arch_size
```

Comment:
We actually try to keep `Expr` immutable, and then avoiding a direct modification of `._size`.
You may want to check the size, and then use a `zeroExtend(arch_size)` or a `[:arch_size]` depending on the result.